src/argilla/client/feedback/dataset/remote/dataset.py

```diff
@@ -480,6 +481,46 @@ def add_records(
         """
         self._records.add(records=records, show_progress=show_progress)
 
+    @allowed_for_roles(roles=[UserRole.owner, UserRole.admin])
+    def search_by(
+        self,
+        vector_name: str,
+        value: Optional[List[float]] = None,
+        record: Optional[RemoteFeedbackRecord] = None,
+        max_results: int = 50,
+    ) -> List[Tuple[RemoteFeedbackRecord, float]]:
```

Comment:
@alvarobartt the same here. It's ok to return `RemoteFeedbackRecord` here?

Reply:
So the `RemoteFeedbackRecord` should just be returned if so, in this case since the search is applied in Argilla, then this is correct!